garage.rb

```diff
@@ -0,0 +1,103 @@
+class Garage
+  TOTAL_PARKING_SPOTS = 200
+  def initialize
+    @daily_revenue = 0
+    @current_parked_status = 0
+  end
+
+  def capacity
+   # Total_Parking_Spots - current_parked_status
+  end
+
+  private
+  def revenue_intake
+    # tally of daily intake
+  end
+
+  def audit_daily_income
+    # reset the daily income to 0
+  end
+end
+
+class Floor
+  FLOORS = 4
+  FIRST_FLOOR_SPOTS = 40
+  SECOND_FLOOR_SPOTS = 50
+  THIRD_FLOOR_SPOTS = 60
+  FOURTH_FLOOR_SPOTS = 60
```

Comment:
Perhaps each `Floor` instance should just manage 1 floor, rather than having info about other floors?